modules/seqexec/web/client/src/main/resources/less/style.less

```diff
@@ -161,16 +161,12 @@ body {
   height: ~"calc(100vh - 44.8em)";
 }
 
-.SeqexecStyles-stepsListPaneWithControls {
+.SeqexecStyles-tableContainer {
   height: ~"calc(100% - 2.8em)";
```

Comment:
I haven't seen this before. Does CSS do this now or is it a Less thing?

Reply:
it is css. Basically it calculates the height as 100% minus something. In this case the height of the area where the buttons are.

It will resize the area when the browser is resized